Engine/CMakeLists.txt

```diff
@@ -84,6 +84,16 @@ add_definitions(-DSTB_TRUETYPE_IMPLEMENTATION)
 add_definitions(-DSTB_IMAGE_WRITE_IMPLEMENTATION)
 
 ### OpenAl
+if(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
```

Comment:
Is this related to openal 1.22, or it's own thing?

Reply:
I copied these from the port of openal-soft for FreeBSD.
https://cgit.freebsd.org/ports/tree/audio/openal-soft/Makefile#n18

Mainly to not use ALSA by default and instead go with OSS.